.travis.yml

```diff
@@ -32,7 +33,9 @@ before_script:
   - make presto-server-rpm.rpm
 script:
   - |
-    if [ -v SHORT_PRODUCT_TEST_GROUP ]; then
+    if [ -v ARTIFACTS ]; then
```

Comment:
looks like this has to go in a separate if-else, otherwise it fails at line 56

Reply:
There was a type ARITFACTS vs ARTIFACTS. Just fixed that.